services/ui-src/src/utils/validation/schemas.ts

```diff
@@ -89,17 +89,27 @@ export const validNumberOptional = () =>
 
 // NUMBER NOT LESS THAN ONE
 export const numberNotLessThanOne = () =>
-  validNumber().test({
+  number().test({
     test: (value) => {
-      return parseFloat(value!) >= 1;
+      if (value) {
```

Comment:
I think an immediate return would be easier to comprehend here, plus shorter variable names:
```
if (!value) return true;

const isValidString = validNAValues.includes(value);
const isGreaterThanOne = parseFloat(value) >= 1;
return isValidString || isGreaterThanOne;
```